modules/foreman/manifests/plugin/fusor_network.pp

```diff
@@ -48,11 +48,6 @@
       peerdns   => true,
     }
 
-    host { $fqdn:
-      comment      => 'created by puppet class foreman::plugin::fusor_network',
-      ip           => $ip,
-      host_aliases => $hostname
-    }
```

Comment:
Removing this from puppet config. Puppet does a really poor job at managing /etc/hosts. The entries are keyed off of the name. It is therefore impossible to purge it and recreate both the IPv4 and IPv6 localhost entries. 

If there is already an old entry for the IP address puppet will not remove it leaving two entries, which is not correct. This could also occur if multiple attempts are made to run the installer.

We can get some really inconsistent results letting puppet manage host resources of I've opted to try to do something a little more intelligent with the the provisioning wizard.